src/styles/components/buttons/styles.less

```diff
@@ -1,19 +1,12 @@
 /* -----------------------------------------------------------------------------
 --------------------------------------------------------------------------------
 --------------------------------------------------------------------------------
-
-components/buttons.less
-
+buttons/styles.less
 --------------------------------------------------------------------------------
 --------------------------------------------------------------------------------
 ----------------------------------------------------------------------------- */
 
-
-
-
-
-
-& when (@button-enabled) {
+& when (@buttons-enabled) {
 
   .go-to-bottom-button {
     bottom: 32px;
```

Comment:
This should be variable driven and change with screen-size.  Let's either resolve here, or note somehow as a TODO.